src/components/base/button/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import propTypes from "prop-types";
+
+import Styles from "./index.module.scss";
```

Comment:
Could we maintain camelCase for naming css variable ?
```suggestion
import styles from "./index.module.scss";
```